FeedStoreChallenge/CD_Cache.swift

```diff
@@ -0,0 +1,47 @@
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(CD_Cache)
+public class CD_Cache: NSManagedObject {}
+
+extension CD_Cache {
+	@NSManaged public var timestamp: Date
+	@NSManaged public var feed: NSOrderedSet
+}
+
+// MARK: Generated accessors for feed
+extension CD_Cache {
+	@objc(addFeedObject:)
+	@NSManaged public func addToFeed(_ value: CD_FeedImage)
+
+	@objc(removeFeedObject:)
+	@NSManaged public func removeFromFeed(_ value: CD_FeedImage)
+
+	@objc(addFeed:)
+	@NSManaged public func addToFeed(_ values: NSSet)
+
+	@objc(removeFeed:)
+	@NSManaged public func removeFromFeed(_ values: NSSet)
+}
+
+extension CD_Cache: Identifiable {}
+
+extension CD_Cache {
+	internal static func find(in context: NSManagedObjectContext) throws -> CD_Cache? {
```

Comment:
`internal` is the default access level, so you don't need to write it explicitly.

It was written explicitly in the first lectures of the program for teaching purposes, but you don't need to do it in your code.

Please, remove all explicit `internal` keywords.
```suggestion
	static func find(in context: NSManagedObjectContext) throws -> CD_Cache? {
```